### PR TITLE
change test.yml and makefile

### PR DESCRIPTION
**Pull request by S1nus**
## Overview

The Code Coverage action appears to be broken, this is an attempt to fix it.

Fix inspired by https://github.com/celestiaorg/celestia-node/pull/2249/files

## Checklist

- [ ] New and updated code has appropriate documentation
- [ ] New and updated code has new and/or updated testing
- [ ] Required CI checks are passing
- [ ] Visual proof for any user facing features like CLI or documentation updates
- [ ] Linked issues closed with keywords

### CODE REVIEW

**Review by MSevey — DISMISSED** [review]
LGTM, yes the failure is because it thinks the coverage went down.
I can admin merge this once other's approve and then update the required CI to not include the remove test step

**Review by Manav-Aggarwal — DISMISSED** [review]

File: .github/workflows/test.yml
Comment (MSevey) [inline .github/workflows/test.yml:40]: sorry, can we actually keep this name? so update https://github.com/rollkit/rollkit/pull/968/files#diff-faff1af3d8ff408964a57b2e475f69a6b7c7b71c9978cccc8f471798caac2c88R25 to be `unit_test`?

Comment (S1nus) [reply .github/workflows/test.yml:40]: fixed

**Review by MSevey — APPROVED** [review]

**Review by nashqueue — APPROVED** [review]

**Pull request merged**
